flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobClientTest.java

```diff
@@ -139,30 +139,35 @@ private static BlobKey prepareTestFile(File file) throws IOException {
 	 * the specified buffer.
 	 * 
 	 * @param inputStream
-	 *        the input stream returned from the GET operation
+	 *        the input stream returned from the GET operation (will be closed by this method)
 	 * @param buf
 	 *        the buffer to compare the input stream's data to
 	 * @throws IOException
 	 *         thrown if an I/O error occurs while reading the input stream
 	 */
 	static void validateGet(final InputStream inputStream, final byte[] buf) throws IOException {
```

Comment:
`s/validateGet/validateGetAndClose/g` - to avoid surprises for the function users :)